### PR TITLE
Specify sign(.) more clearly

### PR DESCRIPTION
**Pull request by dneto0**
Fixes: #3211

### CODE REVIEW

**Review by zoddicus — APPROVED** [review]
lgtm

File: wgsl/index.bs [hunk 1]
```diff
@@ -11846,7 +11846,8 @@ Note: The vec2&lt;f32&gt; case is the same as `unpack2x16float(pack2x16float(e))
     <td>[ALLFLOATINGDECL]
   <tr>
     <td>Description
-    <td>Returns the sign of `e`.
+    <td>Result equals 1.0 when |e| &gt; 0, 0.0 when |e| = 0, and -1.0 when |e| &lt; 0.
```
Comment (alan-baker) [inline wgsl/index.bs:11849]: Why the change from code formatting to variable? Because this isn't an algorithm anymore the e's don't highlight.

Comment (dneto0) [reply wgsl/index.bs:11849]: ugh. thanks for the catch!

Comment (dneto0) [reply wgsl/index.bs]: Fixed.  I also made it a bulleted list of cases, for easier reading.

Screen Shot 2022-07-21 at 10 02 15 AM

**Review by alan-baker — APPROVED** [review]
